lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -0,0 +1,113 @@
+# Check to see how complete the (live) content-store's copies of data are.
+#
+#   pasc = DataHygiene::PublishingApiSyncCheck.new(StatisticsAnnouncement.limit(100))
+#   pasc.add_expectation { |json, model| json.fetch("format") == "statistics_announcement" }
+#   pasc.perform
+#
+# The disadvantages of running against content-store are that you can't inspect
+# things like rendering_app.
+#
+# On the other hand, if you don't trust publishing-api's ability to push to
+# content-store...
+#
+# If we used publishing-api, we could use it's index endpoint to do this all in
+# a single request. The problem is that that endpoint currently doesn't do
+# pagination, so it won't (reliably) handle lots of items...
+module DataHygiene
+  class PublishingApiSyncCheck
+    class Success
+      attr_reader :base_path
+
+      def initialize(base_path:)
+        @base_path = base_path
+      end
+    end
+
+    class Failure
+      attr_reader :base_path, :failed_expectations
+
+      def initialize(options)
```

Comment:
Is there any rationale for choosing kwargs above, but options in the signature here? I'd favour kwargs here too.